README.md

```diff
@@ -0,0 +1,155 @@
+# Vault integration for Python apps
+
+Applications need to be able to obtain sensitive information from Vault
+via its API. Handling this outside of the application is a bit janky; either a
+sidecar process is required, or the app must be periodically killed and
```

Comment:
Do we know the reason for this - is it Vault's issue or Python (library's)? Just curious

Reply:
Just the consequence of managing the secrets outside the app. You could probably get away with not restarting the app if you had the sidecar or daemon managing the secrets write the secrets to disk and then have the app always check the file when it needs them. At that point you're half way to just doing it all inside the app.